rs/rosetta-api/runner/src/lib.rs

```diff
@@ -22,9 +20,18 @@ impl RosettaContext {
         self.port
     }
 
-    pub fn kill(self) {
+    /// Kills the rosetta process and deletes the state directory.
+    /// All traces of the rosetta process will be removed.
+    pub fn kill_rosetta_context(self) {
```

Comment:
If a caller wanted to do what the comment says, wouldn't it be enough to just drop the `RosettaContext` instance? IIUC, if the struct is kept in scope and `kill_rosetta_context()` is called, then the process will be dropped (printing an error if killing it failed), but the `state_directory` will not be removed yet? I couldn't find any callers to this function, so maybe it could be removed?

Reply:
Yes, this serves as a wrapper for the drop so it is more clear what is does. I can remove `kill_rosetta_context`  since it is not used right now.